### DISCUSSION TITLE
Render from USD camera

### DISCUSSION THREAD

**Discussion by yolov** [Autodesk/maya-usd#2759] [General] [answered]
Question on USD cameras -- if you bring a camera with a USD file to Maya, the obvious way to use the camera for rendering is to duplicate it as Maya data (or import just the camera prim(s) with mayaUSDImport) and then use it as a native Maya camera.

Are there other ways to render from a USD camera without importing it?
Are there any other general workflow recommendations here?

Thanks in advance,
Alex

**Comment by neilh-adsk** [comment, marked as answer]
We are working on making the UFE Camera more supported throughout, so you can work with it like a regular camera, Frame a shot etc.  IT was a little more work than we envisaged (isn't it always!!) but the hope is that it can be used mostly as a regular camera in typical workflows.  As to your question, for now I only see a duplicate/clone to Maya camera as a way to really embrace it

Cheers,

Neil

**Comment by yolov** [comment]
Thanks, Neil, that's mighty helpful to know.

Alex
